e2e/plugin-e2e/plugin-e2e-api-tests/as-admin-user/extensions/useExposedComponent.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { test, expect } from '@grafana/plugin-e2e';
+
+const pluginId = 'myorg-componentconsumer-app';
+const exposedComponentTestId = 'exposed-component';
+
+test('should display component exposed by another app', async ({ page }) => {
+  await page.goto(`/a/${pluginId}`);
+  await expect(await page.getByTestId(exposedComponentTestId)).toBeVisible();
+});
```

Comment:
Should we also add an `expect()` to check if the passed in props are working correctly? (E.g. to check if `Hello ${name}!` rendered correctly)

Reply:
Yes good shout! I'll fix